src/aks-preview/azext_aks_preview/_params.py

```diff
@@ -324,6 +327,12 @@ def load_arguments(self, _):
         c.argument('enable_windows_gmsa', action='store_true')
         c.argument('gmsa_dns_server')
         c.argument('gmsa_root_domain_name')
+        c.argument('enable_disk_driver', arg_type=get_three_state_flag())
+        c.argument('disable_disk_driver', arg_type=get_three_state_flag())
+        c.argument('enable_file_driver', arg_type=get_three_state_flag())
+        c.argument('disable_file_driver', arg_type=get_three_state_flag())
+        c.argument('enable_snapshot_controller', arg_type=get_three_state_flag())
+        c.argument('disable_snapshot_controller', arg_type=get_three_state_flag())
```

Comment:
We do not recommend adding two completely opposite parameters repeatedly, because `--enable-disk-driver True` can replace `--disable-disk-driver False`, which will lead to too many similar parameters of CLI, which will increase the learning cost of users

Reply:
@zhoxing-ms so what's the suggestion? only use `--enable-disk-driver`? and `--enable-disk-driver False` looks weird.

Reply:
there are a few parameters following the same way in AKS: e.g.
```
        c.argument('enable_public_fqdn', action='store_true')
        c.argument('disable_public_fqdn', action='store_true')
        c.argument('enable_ahub')
        c.argument('disable_ahub')
        c.argument('enable_azure_rbac', action='store_true')
        c.argument('disable_azure_rbac', action='store_true')
        c.argument('enable_secret_rotation', action='store_true')
        c.argument('disable_secret_rotation', action='store_true')
        c.argument('rotation_poll_interval')
        c.argument('enable_cluster_autoscaler', options_list=[
                   "--enable-cluster-autoscaler", "-e"], action='store_true')
        c.argument('disable_cluster_autoscaler', options_list=[
                   "--disable-cluster-autoscaler", "-d"], action='store_true')
```